lib/routes/huggingface/blog-community.ts

```diff
@@ -0,0 +1,125 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import cache from '@/utils/cache';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/blog-community/:sort?',
+    categories: ['programming'],
+    example: '/huggingface/blog-community',
+    parameters: {
+        sort: {
+            description: 'Sort by trending or recent',
+            default: 'trending',
+            options: [
+                { value: 'trending', label: 'Trending' },
+                { value: 'recent', label: 'Recent' },
+            ],
+        },
+    },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['huggingface.co/blog/community', 'huggingface.co/'],
+        },
+    ],
+    name: 'Community Articles',
+    maintainers: ['yuguorui'],
+    handler,
+    url: 'https://huggingface.co/blog/community',
```

Comment:
```suggestion
    url: 'huggingface.co/blog/community',
```

Reply:
Thanks for your advice, I have fixed it.